DoSomethingArchive/legacy-website
# Campaign  full template refactor

#### What's this PR do?

This PR completely refactors the main campaign template and establishes some new patterns that will be moved over into Neue. It updates the campaign template to use the new `.container` module pattern we have been establishing for content across the site.
#### Where should the reviewer start?

Start by pulling down this branch and testing it locally. You should probably run `drush cc all` once the code is pulled down and before starting to test it.
#### How should this be manually tested?

Test a bunch of different campaigns. Make sure that things are looking correct and close to what they used to look like. Some spacing should be different now that it's more consistent and using the spacing rules defined in the `.container` module. Just make sure no campaigns look borked.
#### Any background context you want to provide?

In the "Drop It Off" stage the spacing for the paragraph and stuff below it is off. Known issue that will be fixed in a different PR to update the use of a markdown field instead of plain text in the CMS. Just FYI.
#### What are the relevant tickets?

Resolves #2307
Resolves #2028
Resolves #2293
Resolves #2425
Resolves #2028
Resolves #1206
Resolves #1177
